AttributeError: 'module' object has no attribute 'indent': textwrap does not have indent in python 2.7

https://travis-ci.org/github/start-jsk/jsk_apc/jobs/675617785

```
Errors << grasp_fusion:make /home/travis/ros/ws_jsk_apc/logs/grasp_fusion/build.make.000.log

Cached Downloading: /home/travis/data/grasp_fusion_lib/grasp_fusion/rosbags/cluttered_tote.bag


Traceback (most recent call last):


  File "/home/travis/ros/ws_jsk_apc/src/jsk_apc/demos/grasp_fusion/ros/grasp_fusion/scripts/install_data.py", line 33, in <module>


    main()


  File "/home/travis/ros/ws_jsk_apc/src/jsk_apc/demos/grasp_fusion/ros/grasp_fusion/scripts/install_data.py", line 16, in main


    md5='1c9e45667d7a225d9bb089d2f45b3682',


  File "/usr/local/lib/python2.7/dist-packages/gdown/cached_download.py", line 123, in cached_download


    download(url, temp_path, quiet=quiet, proxy=proxy, speed=speed)


  File "/usr/local/lib/python2.7/dist-packages/gdown/download.py", line 99, in download


    error = textwrap.indent(error, "\t")


AttributeError: 'module' object has no attribute 'indent'


make[2]: *** [CMakeFiles/grasp_fusion_install_data] Error 1


make[1]: *** [CMakeFiles/grasp_fusion_install_data.dir/all] Error 2


make[1]: *** Waiting for unfinished jobs....


make: *** [all] Error 2
```

--- a/gdown/download.py
+++ b/gdown/download.py
@@ -18,6 +18,16 @@ from .parse_url import parse_url
 CHUNK_SIZE = 512 * 1024  # 512KB
 
 
+if hasattr(textwrap, 'indent'):
+    indent_func = textwrap.indent
+else:
+    def indent_func(text, prefix):
+        def prefixed_lines():
+            for line in text.splitlines(True):
+                yield (prefix + line if line.strip() else line)
+        return ''.join(prefixed_lines())
+
+
 def get_url_from_gdrive_confirmation(contents):
     url = ""
     for line in contents.splitlines():
@@ -96,7 +106,7 @@ def download(url, output=None, quiet=False, proxy=None, speed=None):
         except RuntimeError as e:
             print("Access denied with the following error:")
             error = "\n".join(textwrap.wrap(str(e)))
-            error = textwrap.indent(error, "\t")
+            error = indent_func(error, "\t")
             print("\n", error, "\n", file=sys.stderr)
             print(
                 "You may still be able to access the file from the browser:",

